ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -0,0 +1,169 @@
+'''
+Tests for pforma_io
+'''
+import unittest
+import os
+import sys
+import shutil
+import tempfile
+from mock import patch
+import logging
+from StringIO import StringIO
+from contextlib import contextmanager
+from ph5.clients import ph5tostationxml
+from ph5.utilities import kef2ph5
+from ph5 import logger, ch as CH
+from ph5.core import experiment
+
+
+@contextmanager
+def captured_output():
+    new_out, new_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = new_out, new_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
+
+
+def log_capture_string():
+    capture = StringIO()
+    logger.removeHandler(CH)
+    ch = logging.StreamHandler(capture)
+    logger.addHandler(ch)
+
+    return capture
+
+
+def initialize_ph5(nickname, path='.', editmode=False):
```

Comment:
is '.', current_dir, the best path... this will depend on where it's run. typically this is in the root.
This maybe ok. I'm not sure

Reply:
In the test I wanted to move everything the test creates in tmpdir. So I change the current_dir to to tmpdir in setUp. and create master.ph5 in tmpdir too.